### PR TITLE
rock_02 & rock_03 tweaks

### PR DESCRIPTION
**Pull request by cloudziq**
- removed small seams on textures
- reduce vertex count by merging close ones
- fix the wrong usage of `objskin` command
- adjusted colliders
- added two LOD meshes for each model
  (tris: ~1000 > ~550 > ~140)

### CODE REVIEW

File: rock_03/lod1/obj.cfg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+objload mesh.obj
+objskin mesh ../diff ../mask
+objbumpmap mesh ../norm
+objtricollide collider
+
+mdlscale 50
```
Comment (qreeves) [inline rock_03/lod1/obj.cfg]: No newline at end of file.

File: rock_03/lod2/obj.cfg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+objload mesh.obj
+objskin mesh ../diff ../mask
+objbumpmap mesh ../norm
+objtricollide collider
+
+mdlscale 50
```
Comment (qreeves) [inline rock_03/lod2/obj.cfg]: No newline at end of file.

**Review by qreeves — CHANGES_REQUESTED** [review]
Add newlines at end of two files.

File: rock_03/lod2/obj.cfg (hunk 2 above)
Comment (cloudziq) [reply rock_03/lod2/obj.cfg]: yep, I not double-checked it earlier. Fixed now

**Pull request merged**